### DISCUSSION TITLE
Deformed Mesh Generation

### DISCUSSION THREAD

**Discussion by vansloot** [mdolab/dafoam#221] [General]
Hello,

 I am investigating the effects of rotational speed on blade design/optimal FFD coordinates. I am currently running the Rotor 37 tutorial with modified rotational speeds, and am running into meshing issues when the rotational speed is outside a narrow range. My plan is to perform remeshing on the blade using GMSH when it runs into these problems and to continue the optimization to see if the results change with a new mesh.

Output from the optimization I have  31 timesteps each with a polymesh folder containing only a points.gz file. To the best of my understanding the points are the only aspect of the mesh that change over the course of the optimization ( i.e. cell zones, faces, faceZones, owner, neighbor... in constant/polymesh remain the same). Is my understanding here correct?

I have been trying to use these files to create a deformed mesh at the final timestep.  The closest method I have been able to find so far is using foamMeshToFluent. However, with this command I have only been able to output the original mesh and not any of the deformed ones. Are there any methodologies you would suggest to output  deformed meshes from the dafoam rotor37 tutorial?

Thank you.

**Comment by friedenhe** [comment]
>To the best of my understanding the points are the only aspect of the mesh that change over the course of the optimization ( i.e. cell zones, faces, faceZones, owner, neighbor... in constant/polymesh remain the same). Is my understanding here correct?

Yes

>I have been trying to use these files to create a deformed mesh at the final timestep. The closest method I have been able to find so far is using foamMeshToFluent. However, with this command I have only been able to output the original mesh and not any of the deformed ones. Are there any methodologies you would suggest to output deformed meshes from the dafoam rotor37 tutorial?

You can copy the points.gz from the last optimization to constant/polyMesh/points.gz and then run foamMeshToFluent
